tests/test_graphical_units/img_svg_resources/drawing4.svg

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Are we allowed to include this in our test suite?

Reply:
This one I'm not too sure about. I'll see if I can get a hold of the OP of that logo and ask. I think that's probably the best policy for every one of these files.